src/components/InviteEmail/InviteEmail.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './InviteEmail.css';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './InviteEmail.css'.